src/monocle_apptrace/metamodel/maps/langchain_methods.json

```diff
@@ -46,6 +46,22 @@
         "wrapper_method": "allm_wrapper",
         "output_processor": ["metamodel/maps/attributes/inference/langchain_entities.json"]
     },
+    {
```

Comment:
similar changes to be made for llama index as well ?

Reply:
for llama index azure openai and openai uses same base model , so not required.